### DISCUSSION TITLE
Suggestion: include source images for GIF at doc/json.gif

### DISCUSSION THREAD

**Discussion by dtgriscom-ta** [nlohmann/json#3410] [Ideas]
### Description

The animated GIF shown in the top directory's README.md has a lot of good information in it. However, the animation advances on its own schedule, making it difficult or impossible to pause on slides that interest the viewer. On my Mac, I can download the GIF and open it in Preview, which allows paging through the slides, but I don't know if that's true on other platforms.

Suggestion: include the GIF source slide images were part of the documentation folder. Bonus points if there was an easy way to regenerate the GIF from the slides (perhaps with some common command line tool).

### Reproduction steps

Visit the project home page

### Expected vs. actual results

Some easy way of accessing the individual slides

### Minimal code example

```Shell
n/a
```

### Error messages

```Shell
n/a
```

### Compiler and operating system

MacOS 10.14.6

### Library version

Latest

### Validation

- [X] The bug also occurs if the latest version from the [`develop`](https://github.com/nlohmann/json/tree/develop) branch is used.
- [ ] I can successfully [compile and run the unit tests](https://github.com/nlohmann/json#execute-unit-tests).

**Comment by nlohmann** [comment]
No, there is currently no plan to share the sources for the GIF.

---
*This reply was drafted by [Claude Code](https://claude.com/claude-code) on behalf of @nlohmann, as part of a review of open discussions.*
